src/traces/contourcarpet/index.js

```diff
@@ -15,7 +15,7 @@ ContourCarpet.supplyDefaults = require('./defaults');
 ContourCarpet.colorbar = require('../contour/colorbar');
 ContourCarpet.calc = require('./calc');
 ContourCarpet.plot = require('./plot');
-ContourCarpet.style = require('./style');
```

Comment:
continuing to 🌴  contour/contourcarpet... this also fixes some bugs such as:
- contourcarpet failed with `contours.coloring='lines'`
- if the first contour wasn't shown (if you give an explicit contour start value lower than what the data would be) the colors would have been wrong. For contour this was fixed in #1309 (the key is to use `d.level` rather than `i` in coloring the contour lines/fills) and tested in `contour_edge_cases`.

@etpinard do you think it's sufficient to rely on the contour tests for contourcarpet too - since it's using the same code now - or would you like me to add contourcarpet tests of these cases?